src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -249,6 +257,10 @@ private static boolean isTinymceStyleAttribute(Node attribute) {
         return checkForAttributeWithSpecificValue(attribute, "id", "mceDefaultStyles");
     }
 
+    private static boolean isTinymceEditorAttribute(Node attribute) {
+        return checkForAttributeWithSpecificValue(attribute, "class", "content-editor");
```

Comment:
Maybe better if the selector is `mce-content-body`? The probability of us self-authoring something with class `mce-*` is close to zero, but `content-editor` sounds somewhat probable.